src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -804,6 +806,56 @@ public Response deleteProject(
         }
     }
 
+    @POST
+    @Path("/batchDelete")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Deletes a list of projects specified by their UUIDs",
+            description = "<p>Requires permission <strong>PORTFOLIO_MANAGEMENT</strong></p>"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "204", description = "Projects removed successfully"),
+            @ApiResponse(responseCode = "207", description = "Access is forbidden to the projects listed in the response"),
```

Comment:
Not sold on this status code. Some surface-level research leads me to believe using this outside of WebDAV is not intended: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/207

Wouldn't it be better to atomically fail the request if deletion of one or more projects failed?

Reply:
Fair point, the 207 indeed seems to be reserved for WebDAV. The reason for implementing it this way was based on the outcome of [this](https://github.com/DependencyTrack/dependency-track/pull/3407#discussion_r1523485369) discussion, where the suggestion by [sebD](https://github.com/sebD) was to return a 204 with a body containing the list of inaccessible projects. The 207 was my suggestion, and since I got a thumbs up I thought I would do it the same way in this PR. I see a value in letting the client know which projects that are inaccessible, so I'm leaning towards going with sebD's suggestion instead of just returning a 401. Would that be an acceptable solution?

Reply:
Ok, I'll look into using ProblemDetails since that will make it consistent with how failed tag deletion is handled. Thanks for the tip!